demos/multi_channel/object_detection_demo_yolov3/CMakeLists.txt

```diff
@@ -59,7 +59,7 @@ if(MULTICHANNEL_DEMO_USE_TBB)
 endif()
 
 target_link_libraries(${TARGET_NAME}
-    ${InferenceEngine_LIBRARIES} gflags ${OpenCV_LIBRARIES} ngraph::ngraph multi_channel_common)
+        ${TBB_IMPORTED_TARGETS} ${InferenceEngine_LIBRARIES} ngraph::ngraph gflags ${OpenCV_LIBRARIES} multi_channel_common)
```

Comment:
Dear @Wovchena  in an ideal world you're correct. This should be fixed in dldt. However this recipe is highly reliable and works every time and we need an urgent solution ASAP. Do you accept the patch ? Please let me know definitively.

Reply:
@ilya-lavrenov, @asomsiko, @ilyachur, can you confirm the problem can't be resolved on IE side quickly?


As for the problem definition, there are some setups when demos build fails because of missing tbb and ngraph. And simple export of `InferenceEngine_DIR`, `TBB_DIR`, `OpenCV_DIR` and `ngraph_DIR` doesn't work without cmake modifications. @shubha-ramani, am I correct?

Reply:
@Wovchena  you are correct. With this pull request, all user has to do is set those 4 environment variables you mention - and the demos build will **_succeed every time._** And this is true regardless of whether the openvino build is open source, gitlab, released openvino, cross-compiled or whatever. That is the beauty and elegance of this pull-request.

Reply:
I don't understand at the current moment why do you need TBB and nGraph. Demos don't touch it, Inference Engine has a private dependency on these components. In general if you use the right environment all should work.
Am I wrong?